events/iq_board.py

```diff
@@ -0,0 +1,94 @@
+import discord
+from discord.ext import commands
+import utils.logging as log
+import utils.colours as colour
+import utils.values as value
+import utils.database as sql
+from utils.helpers import *
+
+
+class IQBoard(commands.Cog):
+    def __init__(self, client):
+        self.client = client
+        self.iq_emoji = "<:WeSmart:682352802202386472>"
+        self.iq_requirement = 7
+
+    @commands.Cog.listener()
+    async def on_ready(self):
+        self.iq_channel = self.client.get_channel(-1)  # TODO: Create channel and change ID
```

Comment:
Seems ok. Not sure if it would make more sense to put this into ready.py or not. To be honest, it might be an idea to start doing this for other commands (for example setting self.up value inside the ping command with its own on_ready event listener to make sure everything is kept separate).

Reply:
I'm more for moving each value in the file that's most relevant to it, as you said for self.up should be moved in the ping command and so on.
I don't see any obvious advantages to this other than avoiding possible breakage if we reload a cog during a raffle, poll or ping, even if rare I see small problems appearing in the future.

Reply:
There shouldn't be any immediate breakage with values if they're all initialised in main.py from reloading cogs